src/cli/slatkin-maddison.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const phylotree = require('../../build/phylotree.js');
+const commander = require('commander');
+const pd = require("pretty-data").pd;
+
+commander.option(
+    '-n --newick <newick>',
+    'Input newick file'
+  ).option(
+    '-i --iterations [iterations]',
+    'Number of bootstrap iterations (default is 10000)',
+    10000
+  ).option(
+    '-s --seed [seed]',
+    'Seed for random number generator',
+    1
+  ).option(
+    '-r --regex <regex> [otherRegexes...]',
```

Comment:
Regular expressions can be cumbersome to use and not entirely user friendly. A CSV would be better.